src/shared/menu/helpers.ts

```diff
@@ -243,6 +244,13 @@ export function makeMenuButton(
         button.classList.add(...cssClasses);
     }
 
+    const title = typeof content === "string" ? content : content.title;
+    const helpText = typeof content === "string" ? null : content.helpText;
+
+    if (helpText) {
+        button.dataset.sTooltipHtmlTitle = `<p class="m0">${title}</p><p class="fs-caption fc-black-600 m0">${helpText}</p>`;
```

Comment:
Nit: spacing as-is looks a little tight in the context of the padded popover. I suggest adding a little space between lines.
```suggestion
        button.dataset.sTooltipHtmlTitle = `<p class="mb4">${title}</p><p class="fs-caption fc-black-600 m0">${helpText}</p>`;
```